docs/contribution.rst

```diff
@@ -0,0 +1,128 @@
+======================
+Contributing to ecl2df
+======================
+
+Contributing to ecl2df is easiest on Linux computers. Windows has not been
+tested, and for Mac you will have to compile OPM yourself.
+
+Getting started as a developer
+------------------------------
+
+Clone the master repository from Github::
+
+  git clone https://github.com/equinor/ecl2df
+
+After cloning, change to the newly created ``ecl2df`` directory. The commands
```

Comment:
Instead of cloning from master, it is much better to:

- The user shall always fork using the fork button in github (simpler) if contributions is planned
- Then git clone git@github.com:theuser/ecl2df
- cd ecl2df; git remote add upstream git@github.com:equinor/ecl2df

https://github.com/MarcDiethelm/contributing/blob/master/README.md
https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#forking